docs/BREAKINGCHANGES.md

```diff
@@ -294,3 +294,19 @@ With this change, W3C extended log format is supported.
 
 Remove the `BuildVersion` property from `$PSVersionTable`. This property was tied to the Windows build version.
 Instead, we recommend that you use `GitCommitId` to retrieve the exact build version of PowerShell Core.
+
+### Changes to Web Cmdlets
+
+The underlying .NET API of the Web Cmdlets has been changed to `System.Net.Http.HttpClient`. This change provides many benefits.
```

Comment:
Please use [semantic line breaks](http://sembr.org/)

Reply:
I think the end of the first sentence should be:
 due to differences between .Net and CoreFx.

Reply:
Maybe - .Net Framework and .Net Core.

Reply:
But that change actually doesn't have anything to do with .NET Core and .NET Framework?  Unless I'm mistaken, WebRequest still exists in .NET Core. The decision to move to HttpClient appeared to be more about performance, code simplification, and it being a better fit for these cmdlets and nothing about any particular .NET Core limitations.

As for semantic line breaks.. after reading that document I have no clue what I need to do there. Sentences on separate lines?

Reply:
@markekraus when the webcmdlets were initially ported, only dotnetcore 1.x was available and only HttpClient was available.  This has changed with dotnetcore 2.0 and we didn't go back and revert the code change (remember we only moved to dotnetcore 2.0 for RC1).  Maybe it doesn't matter to bring up history since it's no longer applicable.

Reply:
Ok, good to know. What should I say here then?

Reply:
>remember we only moved to dotnetcore 2.0 for RC1 

We moved to 2.0.1 in RC1 and to 2.0.0-preview in april 2017.

Reply:
>remember we only moved to dotnetcore 2.0 for RC1

Move to the official .NET Core 2.0  #4603 - august 2017.
Move powershell to .NET Core 2.0 (preview)  #3556  - april 2017.

Reply:
Sorry, meant beta

Reply:
@markekraus upon reflection, let's just leave it the way you wrote it except add a line break :)

Reply:
fixed.